<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-66CRK13" failures="12" tests="15" name="Test" time="8.512" errors="0" timestamp="2023-05-04T21:42:33 ART">
  <testcase classname="saucedemo_tests.github.Login_Chrome" name="login_Credentials_Lockout" time="0.372">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Chrome.login_Credentials_Lockout(Login_Chrome.java:108)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Lockout -->
  <testcase classname="saucedemo_tests.github.Login_Chrome" name="login_Credentials_None" time="0.181">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Chrome.login_Credentials_None(Login_Chrome.java:180)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_None -->
  <testcase classname="saucedemo_tests.github.Login_Chrome" name="login_Credentials_Ok" time="0.556"/>
  <testcase classname="saucedemo_tests.github.Login_Chrome" name="login_Credentials_Password_Wrong" time="0.174">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Chrome.login_Credentials_Password_Wrong(Login_Chrome.java:159)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Password_Wrong -->
  <testcase classname="saucedemo_tests.github.Login_Chrome" name="login_Credentials_Username_Wrong" time="0.393">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Chrome.login_Credentials_Username_Wrong(Login_Chrome.java:129)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Username_Wrong -->
  <testcase classname="saucedemo_tests.github.Login_Edge" name="login_Credentials_Lockout" time="0.329">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Edge.login_Credentials_Lockout(Login_Edge.java:108)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Lockout -->
  <testcase classname="saucedemo_tests.github.Login_Edge" name="login_Credentials_None" time="0.179">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Edge.login_Credentials_None(Login_Edge.java:180)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_None -->
  <testcase classname="saucedemo_tests.github.Login_Edge" name="login_Credentials_Ok" time="0.572"/>
  <testcase classname="saucedemo_tests.github.Login_Edge" name="login_Credentials_Password_Wrong" time="0.128">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Edge.login_Credentials_Password_Wrong(Login_Edge.java:159)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Password_Wrong -->
  <testcase classname="saucedemo_tests.github.Login_Edge" name="login_Credentials_Username_Wrong" time="0.305">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Edge.login_Credentials_Username_Wrong(Login_Edge.java:129)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Username_Wrong -->
  <testcase classname="saucedemo_tests.github.Login_Firefox" name="login_Credentials_Lockout" time="0.333">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Firefox.login_Credentials_Lockout(Login_Firefox.java:108)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Lockout -->
  <testcase classname="saucedemo_tests.github.Login_Firefox" name="login_Credentials_None" time="0.271">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Firefox.login_Credentials_None(Login_Firefox.java:180)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_None -->
  <testcase classname="saucedemo_tests.github.Login_Firefox" name="login_Credentials_Ok" time="0.406"/>
  <testcase classname="saucedemo_tests.github.Login_Firefox" name="login_Credentials_Password_Wrong" time="0.257">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Firefox.login_Credentials_Password_Wrong(Login_Firefox.java:159)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Password_Wrong -->
  <testcase classname="saucedemo_tests.github.Login_Firefox" name="login_Credentials_Username_Wrong" time="0.339">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at saucedemo_tests.github.Login_Firefox.login_Credentials_Username_Wrong(Login_Firefox.java:129)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Credentials_Username_Wrong -->
</testsuite> <!-- Test -->
